tool/multiplayer.re

```diff
@@ -150,6 +141,38 @@ static mut MULTIPLAYER_COMPONENT = Component {
 //                }
             }
         }
+        match MULTIPLAYER_STATE.ready_state {
+            ReadyState::StartingAt(ts) => {
+                let time = current_time_millis();
+                if time >= ts {
+                    print(f"starting synchronized new game at {time} (expected {ts})");
+                    Tas::restart_game();
+                    MULTIPLAYER_STATE.ready_state = ReadyState::NotReady;
+                }
+                let viewport = Tas::get_viewport_size();
+                let mut new_time = f"{(ts - time) / 1000:1}.{(ts - time) % 1000:03}";
+                let msg = "Starting new game in...";
+                let mut text_size = Tas::get_text_size(new_time, 1.);
```

Comment:
No need for mut. Just shadow the original variable. No modification is needed. (Makes it easier to reason about the code -- I know that `text_size` won't be changed).